pandas/core/strings.py

```diff
@@ -1180,35 +1180,74 @@ def str_slice(arr, start=None, stop=None, step=None):
 
 def str_slice_replace(arr, start=None, stop=None, repl=None):
     """
-    Replace a sliced string.
-
-    Replace a slice of each string in the Series/Index with another
-    string.
+    Replace a positional slice of a string with another value.
 
     Parameters
     ----------
-    start : int or None
-        Left edge index.
-    stop : int or None
-        Right edge index.
-    repl : str or None
-        String for replacement.
+    start : int, optional
+        Left index position to use for the slice. The default of None
```

Comment:
General question: do we more want to use the phrase of "If not specified" instead of "The default of None" if we say something is optional? 
On the one hand I think this can be clearer as that is what you do in practice, you don't specify it. But on the other hand of course the signature shows None.

Reply:
Yeah, if not specified is probably best.

Reply:
Went with `if not specified (None)`

Reply:
Ah, `If not specified (None)`, is a good combination of both, we should use that pattern more